### ISSUE TITLE
Probe lives in the past and won't get the correct time.

### ISSUE THREAD

**Issue by DutchessNicole** [jsdelivr/globalping-hwprobe#50]
Well that was a misclick.

Either way, I received a hardware probe in the mail today, and while it does run and I can access the logs using SSH, the probe lives in 2022 and won't update the time.

```
[2022-04-28 17:49:33] [ERROR] [api:error] Connection to API failed: certificate is not yet valid
```

Of course without a reasonable time none of the connections will succeed.
Is there a requirement for the probe to reach a certain ntp source? I do have an ntp server running on my local network and the dhcp option for ntp servers is available on the network the probe is on.
However the probe does not seem to access the local ntp server, so wrong time.

**Comment by jimaek** [comment]
Hmm pretty sure we do sync time from pool.ntp.org. https://github.com/jsdelivr/globalping-hwprobe/blob/master/meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-firstBoot.sh#L11

Any chance you're blocking it? e.g. a firewall rule on the router

**Comment by DutchessNicole** [comment]
Absolutely. There's no reason for devices to look on the internet when there's a perfectly fine NTP server right on the local network.

however with tcpdump I found out that while the probe was requesting a timesync, even with the local NTP server, there was no reply coming. So I had to add the new network to ntpsec. That is something I normally don't have to do since everything already lives on defined networks.

And with that after about 10 minutes the probe suddenly started living in the present.
